common/tools/dev-tool/src/commands/samples/checkNodeVersions.ts

```diff
@@ -67,20 +67,42 @@ async function cleanup(
   await deleteDockerImages(dockerImageNames);
 }
 
+function findSamplesDir(samplesDir: string, rootDir: string): string {
+  const dirs = [];
+  for (const file of fs.readdirSync(samplesDir)) {
+    const stats = fs.statSync(path.join(samplesDir, file));
+    if (stats.isDirectory()) {
+      const matchResult = file.match(/^v([0-9]*.*)$/);
```

Comment:
It's of no great importance, but collecting the full directory names instead of capturing from the RegExp may be simpler.

Reply:
good point!, done.